packages/client/src/utils/constants.ts

```diff
@@ -92,6 +92,7 @@ export const JURISDICTION_TYPE: { [key: string]: string } = {
   MUNICIPALITY: 'Municipality',
   CITY_CORPORATION: 'City Corporation',
   UPAZILA: 'Upazila',
+  STATE: 'Province',
   DISTRICT: 'District',
   DIVISION: 'Division'
 }
```

Comment:
@Zangetsu101 looks like we have inherited some technical debt here.  This JURISDICTION_TYPE property should be used to return a translation ID that contains the text, otherwise this will always be in English.

Reply:
I think we should also rename DIVISION, CITY_CORPORATION, UNION, MUNICIPALITY and UPAZILA.  These are bespoke jurisdiction types for Bangladesh.  Lets use STATE, CITY, LOCATION_LEVEL_5, LOCATION_LEVEL_4, LOCATION_LEVEL_3 respectively

Reply:
Okay, will update these.

Reply:
@euanmillar Are location names localized even?  From what I see they are used exactly as returned from the resources package.

Reply:
@Zangetsu101 The names themselves are not localised, but the design team wanted to append this label so that you could differentiate if the district and state has the same name.  So what I am asking for is for us to use localised labels after the name which isnt localised.